### ISSUE TITLE
2 programs using my joycon at once?

### ISSUE THREAD

**Issue by BOBBIESWAG** [mfosse/JoyCon-Driver#93]
Is it possible for 2 programs to be using my joycon? I really want to play games like fortnite with this but since the driver is using my joycon it is unresponsive when trying to use JoyToKey to map the rest of the controls. Is it possible to get around this with this game and others too? I can use Dolphin with it so why wouldnt it work with other games? -Jacob

**Comment by JMSoundwave** [comment]
I use a program called REM4P it only works through ps4 remote play. it converts mouse and keyboard to ps4 input. Also Rem4p has a feature to disable all other mouse input during game. This is the kind of concept i think you mean correct me if  im wrong. I assume your on pc though?

**Comment by BOBBIESWAG** [comment]
Yeah i am, i want to be able to use the gyro mouse from the driver and map the keyboard controls so i can use them both at the same time to play various games.

**Comment by JMSoundwave** [comment]
It depends on the game I've had both good and bad luck with various games it just comes down to tricking the computer into thinking it's real controller.
